src/Paket.VisualStudio/Commands/AddPackageProcess.cs

```diff
@@ -1,47 +1,48 @@
-﻿using Microsoft.FSharp.Control;
+﻿using System.Reactive;
+using Microsoft.FSharp.Control;
 using Microsoft.FSharp.Core;
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Reactive.Linq;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 using System.Windows.Forms;
+using Paket.VisualStudio.Commands.PackageGui;
 
 namespace Paket.VisualStudio.Commands
 {
     public class AddPackageProcess
     {
+        public static Task<string[]> SearchPackagesByName(string name)
+        {
+           return FSharpAsync.StartAsTask(NuGetV3.FindPackages(FSharpOption<Paket.Utils.Auth>.None, Constants.DefaultNugetStream,
+                    name, 1000),
+                FSharpOption<TaskCreationOptions>.None,
+                FSharpOption<CancellationToken>.None);
```

Comment:
no worries. I merged it anyways since it's already much better than before. Please send new PR if you want to improve it ;-)